<?xml version="1.0" encoding="ISO-8859-15"?>

<launch>
  <arg name="model_path"   default="$(find ffld_nx)/data/models/polaris_noparts.txt" />
  <arg name="cam_topic"    default="/sync/cam0/image_raw" />
  <arg name="interval"     default="5"      />
  <arg name="resize_val"   default="1.0"    />
  <arg name="rate"	   default="-1.0"   />
  
  <node name="object_detector" pkg="ffld_ros" type="detect_object_node"
        output="screen">
    <param name="model_name" value="$(arg model_path)"/> 
    <param name="overlap" value="0.5" />    <!-- Minimum overlap in non maxima suppression (default 0.5)  -->
    <param name="threshold" value="-0.5" />  <!-- Minimum detection threshold (default -1) -->
    <param name="interval" value="$(arg interval)" />
    <param name="cam_topic" value="$(arg cam_topic)" />
    <param name="resize_val" value="$(arg resize_val)" /> 
    <param name="rate" value="$(arg rate)" /> 
  </node>
</launch>

